components/o-header/src/scss/features/_drawer.scss

```
@@ -71,30 +71,58 @@
 	//
 	.o-header__drawer-tools {
 		overflow: hidden;
-		padding: ($_o-header-drawer-padding-y * 1.5) 0 $_o-header-drawer-padding-y $_o-header-drawer-padding-x;
+		padding: ($_o-header-drawer-padding-y * 2.5) 0 0 $_o-header-drawer-padding-x;
 		background-color: _oHeaderGet('drawer-tools-background');
 		color: _oHeaderGet('drawer-tools-text');
+		display: flex;
+		flex-direction: row;
+		justify-content: space-between;
+		align-items: center;
+
+		.o-header__drawer-edition-switcher {
+			order: 2;
+			display: flex;
+
+			.o-header__drawer-menu-link {
+				padding-left: 12px;
+				text-decoration: underline;
+			}
+
+			.o-header__drawer-current-edition {
+				font-weight: 600;
+				color: var(--Black-Tints-Black-60, #66605C);
```

This custom property doesn't exist.

```suggestion
				color: oColorsByName('black-60');
```